### PR TITLE
Bugs connector encoding change

### PR DESCRIPTION
**Pull request by JeonHyungJin**
Bugs connector which i added has an encoding problem, because it contains Korean.
So i changed bugs.js file`s encoding in connector repository.

### CODE REVIEW

**Comment by JeonHyungJin** [comment]
I missed an encoding problem.... Sorry.. haha

File: src/connectors/bugs.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ Connector.durationSelector = '.finish';
 
 Connector.isPlaying = () => {
 	let btn = $('.btnPlay').first();
-	return btn.html() === '�Ͻ����'; // if text '�Ͻ����' in button, it means 'pause' because music is playing.
+	return btn.html() === '일시정지'; // if text 'ÀÏ½ÃÁ¤Áö' in button, it means 'pause' because music is playing.
```
Comment (inverse) [inline src/connectors/bugs.js]: Can we remove the comment since I dont think it brings any value.

Comment (JeonHyungJin) [reply src/connectors/bugs.js]: Yes. you can remove that comment!

**Comment by JeonHyungJin** [comment]
I removed that comment!

**Comment by JeonHyungJin** [comment]
Thanks!!!!

**Pull request merged**